### ISSUE TITLE
New logo idea

### ISSUE THREAD

**Issue by jglovier** [mislav/movieapp#11]
@mislav I was thinking about a visual refresh for the app when you showed it to me, and the first question I wondered is what would a logo update look like (since the branding should lay a foundation for the rest of the visuals).

Cam up with a couple concepts based on the idea of a movie screen and seats in a theater.

What do you think? Reactions?

<img width="557" alt="screen shot 2015-11-16 at 10 09 49 am" src="https://cloud.githubusercontent.com/assets/1319791/11185326/3a7fcacc-8c4a-11e5-8832-f869e77ac8ee.png">

<img width="556" alt="screen shot 2015-11-16 at 10 10 55 am" src="https://cloud.githubusercontent.com/assets/1319791/11185353/5aabc9ae-8c4a-11e5-9af3-c49f5f54d8f3.png">

**Comment by mislav** [comment]
Nice :smile: This was [on our roadmap](https://www.pivotaltracker.com/n/projects/449605).

I like the direction, but the main problem is that we never wanted to call it `movi.im` in the first place. That's the only short and cutesy domain we could come up with at the moment. But we don't have other ideas. So if we're coming up with some logo, can it at least say just "movi" (even though it doesn't mean anything) since the TLD isn't important?

I like the 1st variant, but it's really not clear that the bottom pattern are the movie theatre seats. It looks more like a chainsaw of some kind. From the 2nd variant it's more apparent, but I like the white background less.

**Comment by jglovier** [comment]
>So if we're coming up with some logo, can it at least say just "movi" (even though it doesn't mean anything) since the TLD isn't important?

![thumbs-up-80s-kid](https://cloud.githubusercontent.com/assets/1319791/11217421/2fea8c58-8d1e-11e5-8552-1f49efa4fcc8.gif)

>I like the 1st variant, but it's really not clear that the bottom pattern are the movie theatre seats. It looks more like a chainsaw of some kind. From the 2nd variant it's more apparent, but I like the white background less.

Yeah, the first ver plays heavily into the white space. Could also help if it was on a dark background. I'll play around with it a little more.

**Comment by jglovier** [comment]
Here's a few new variants. I feel like the top left version has the strongest typography and will work best at smaller sizes, too.

<img width="888" alt="screen shot 2015-11-21 at 6 36 21 pm" src="https://cloud.githubusercontent.com/assets/1319791/11321063/f1d11fd6-907e-11e5-866c-7280fc02795c.png">

**Comment by mislav** [comment]
I like both top variants :)

**Comment by parveen-sishodiya-git** [comment]
Yes, both top variants are looking great but top leftmost is greatest and looking like a brand logo :)
